.travis.yml

```diff
@@ -4,7 +4,7 @@
 dist: trusty
 language: rust
 rust:
-  - nightly
+  - nightly-2018-04-08
```

Comment:
Hm, this seems dubious but otherwise okay.

Reply:
Yeah let me fix that, just spotted too